src/components/Item/AppItem/Item.js

```diff
@@ -39,19 +39,31 @@ const AppItem = ({ dashboardMode, item, itemFilters }, context) => {
         )
     }
 
+    const hideTitle =
+        appDetails?.settings?.dashboardWidget?.hideTitle &&
+        dashboardMode != EDIT
```

Comment:
Oops, missed this: use strict equality checks: `!==`

Reply:
fixed